src/github_changelog/conventional.clj

```diff
@@ -49,14 +88,15 @@
 
 (defn parse-pull [config {:keys [title] :as pull}]
   (if-let [pull-id (parse-revert config pull)]
-    {:revert-pull pull-id
+    {:revert-pull  pull-id
      :pull-request pull}
+    ;; TODO - when-let?
     (if-let [[_ type scope subject] (re-find angular-pattern title)]
```

Comment:
I see no reason why this can't be a when-let.

Reply:
Yes, I believe so. I put that as a TODO because the tests were failing and I could not verify the correctness of the function.